flybywiresim/a32nx
# fix(nd): keep to-wpt-state on mode switch

## Summary of Changes
The ND would lose the to-waypoint-identifier when switching between modes until the next WPT change.

## Screenshots (if necessary)



## References
n/a

## Additional context
This pulls up the wpt/vor/ils information and ident state to the ND so it is not lost on mode changes. Before this was received in the ToWaypointIndicator from a CoherentEvent (opposed to simvars like the other data), this state is now passed in as a prop in order to not lose it on mode switch (and ToWaypointIndicator recreation).

An alternative would be to set the ident as a simvar and receive it from there, but I'm not aware of the implications since I'm not to firm with flight simulator development yet.


## Testing instructions
* spawn c/d or on rwy
* enter a wpt
* switch between ND modes
* check to-waypoint-identifier is still displayed


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
